[2024-10-11T19:06:41.528+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2024-10-11T19:06:41.873+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2024-10-04T11:15:00+00:00 [queued]>
[2024-10-11T19:06:41.925+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2024-10-04T11:15:00+00:00 [queued]>
[2024-10-11T19:06:41.927+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2024-10-11T19:06:41.995+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): first_task> on 2024-10-04 11:15:00+00:00
[2024-10-11T19:06:42.011+0000] {standard_task_runner.py:57} INFO - Started process 65 to run task
[2024-10-11T19:06:42.021+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2024-10-04T11:15:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/qwe.py', '--cfg-path', '/tmp/tmpclsqvgcz']
[2024-10-11T19:06:42.023+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask first_task
[2024-10-11T19:06:42.349+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-10-11T19:06:42.483+0000] {task_command.py:410} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2024-10-04T11:15:00+00:00 [running]> on host 81d81e8b5952
[2024-10-11T19:06:42.591+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2024-10-11T19:06:42.882+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dan' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T11:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-04T11:15:00+00:00'
[2024-10-11T19:06:42.884+0000] {logging_mixin.py:149} INFO - Hello world
[2024-10-11T19:06:42.885+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-10-11T19:06:42.920+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, execution_date=20241004T111500, start_date=20241011T190641, end_date=20241011T190642
[2024-10-11T19:06:43.045+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-11T19:06:43.170+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
